sentry-android-replay/src/main/java/io/sentry/android/replay/ReplayIntegration.kt

```diff
@@ -322,17 +320,16 @@ public class ReplayIntegration(
             return
         }
 
-        recorder?.stop()
-
-        // refresh config based on new device configuration
-        val recorderConfig = recorderConfigProvider?.invoke(true) ?: ScreenshotRecorderConfig.from(context, options.sessionReplay)
-        captureStrategy?.onConfigurationChanged(recorderConfig)
-
-        recorder?.start(recorderConfig)
-        // we have to restart recorder with a new config and pause immediately if the replay is paused
-        if (lifecycle.currentState == PAUSED) {
-            recorder?.pause()
+        captureStrategy?.stop()
```

Comment:
hm, I'm thinking this is too destructive for this case, or am I missing something? Because `stop` here will reset everything and start a new replay pretty much